tests/testthat/test-standardErrorOfMeasurement.R

```diff
@@ -0,0 +1,125 @@
+
+
+# test the sem functions for ordinal data
+
+options <- list(
+  .meta = list(
+    variables = list(
+      hasTypes = TRUE,
+      shouldEncode = TRUE
+    )
+  ),
+  anova = TRUE,
+  combinedPointPlot = TRUE,
+  feldt = TRUE,
+  feldtNumberOfSplits = "5",
+  hideTable = FALSE,
+  histogramCounts = FALSE,
+  irt = TRUE,
+  keats = FALSE,
+  lord = FALSE,
+  lord2 = FALSE,
+  lord2NumberOfSplits = "5",
+  minimumGroupSize = 10,
+  mollenkopfFeldt = TRUE,
+  mollenkopfFeldtNumberOfSplits = "5",
+  mollenkopfFeldtPolyDegree = 3,
+  plotHeight = 320,
+  plotWidth = 480,
+  pointPlots = TRUE,
+  reliabilityValue = 0.5,
+  thorndike = TRUE,
+  userReliability = FALSE,
+  variables = list(
+    types = c("ordinal", "ordinal", "ordinal", "ordinal", "ordinal"),
+    value = c("asrm_1", "asrm_2", "asrm_3", "asrm_4", "asrm_5")
+  )
+)
+
+results <- runAnalysis("standardErrorOfMeasurement", data = "asrm.csv", options = options)
+# results <- runAnalysis("standardErrorOfMeasurement", dataset = Bayesrel::asrm, options = options, makeTests = TRUE)
+
+
+
+# binary data
+options <- analysisOptions("standardErrorOfMeasurement")
```

Comment:
please change this to directly specify the options list, jaspTools cannot handle this.